docs/de_DE/budget.html

```diff
@@ -0,0 +1,90 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+
+    <title>Budget</title>
+    <link href="../master.css" rel="stylesheet" type="text/css">
+    <script src="../js/translator.js"></script>
+    <script src="../js/translations.js"></script>
+    <script src="../js/toc.js"></script>
+    <script src='http://ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js'></script>
+    <script src="../js/back_to_top.js"></script>
+    <link href="../back_to_top.css" rel="stylesheet" type="text/css">
+</head>
+
+<body dir="ltr" lang="de-DE" onload="generateTOC(document.getElementById('toc'));">
+    <a class="i18n" id="go_home" href="./index.html" style="font-weight: bold">Home</a><br>
+    <a href="#" class="back-to-top"><img src="../img/backtotop.png" alt="Back to Top" /></a>
+    <!--Smooth Back to Top Button End-->
+    <h1 class="i18n">Budgeting</h1>
+    <div id="toc">
+        <!-- ToC generated by Javascript -->
+    </div>
+    <hr>
+    <p class="i18n" id="bud_1">
+        Mit <strong>MMEX</strong> können sie Jahres- oder Monatsbudgets erstellen.
+    </p>
+    <p class="i18n" id="bud_2">Damit können sie ihre tatsächlichen Ausgaben mit ihren Budgets vergleichen.</p>
+    <p class="i18n" id="bud_3">Budgets können in Kalenderjahren oder Finanzjahren dargestellt werden.</p>
+    <p class="i18n" id="bud_4">
+        Verwenden Sie das Optionsmenü um den Start des Finanzjahres und andere dauerhafte Einstellungen zu treffen.
+    </p>
+    <p class="i18n" id="bud_5">
+        The <strong>View&rarr;Budgets: As Financial Years</strong> can also be used to switch
+        between the 2 views
+    </p>
+    <hr>
+    <h2 class="i18n">Creating New Budgets</h2>
+    <p class="i18n" id="bud_6">
+        To set up a budget right click on the 'Budgeting' tree node and add a
+        budget month or year.
+    </p>
+    <div class="row">
+        <div class="col-xs-1"></div>
+        <img class="shadow" id="budget_editor" alt='Budget Editor' src='budget_editor.png'>
+    </div>
+    <hr>
+
+    <h2 class="i18n">Editing of Budget Categories</h2>
+    <p class="i18n" id="bud_7">
+        Once a year has been added, select the year to display the Budget
+        Setup.
+    </p>
+    <p class="i18n" id="bud_8">
+        Each category can then be edited by double clicking the selected
+        category.
+    </p>
+    <div class="row">
+        <div class="col-xs-1"></div>
+        <img class="shadow" id="budget_entry_details" alt="Bugget Entry Detail" src="budget_entry_details.png">
+    </div>
+    <br>
+    <p class="i18n" id="bud_9">This is done for all subsequent categories.</p>
+    <hr>
+
+    <h2 class="i18n">Budget Setup</h2>
+    <p class="i18n" id="bud_10">
```

Comment:
```suggestion
    <p>
```

Reply:
It will be fixed automaticly ASAP as new translated strings added to de_DE locale.